impl/src/test/java/io/kroxylicious/testing/kafka/KafkaClusterTest.java

```diff
@@ -129,9 +131,11 @@ public void kafkaClusterZookeeperModeWithAuth() throws Exception {
 
     @Test
     public void kafkaClusterKraftModeSASL_SSL() throws Exception {
+        createClientCertificate();
         try (var cluster = KafkaClusterFactory.create(KafkaClusterConfig.builder()
                 .testInfo(testInfo)
-                .keytoolCertificateGenerator(keytoolCertificateGenerator)
+                .brokerKeytoolCertificateGenerator(brokerKeytoolCertificateGenerator)
+                .clientKeytoolCertificateGenerator(clientKeytoolCertificateGenerator)
```

Comment:
I would have a separate test(s) for the SSL client auth.  For the other SSL tests, the client should not have a keystore, only a trust store.